docs/index.md

```diff
@@ -12,7 +12,7 @@ VoteAmerica provides a handful of products for other organizations to use:
 * [Civic Data API](api/index.md): 80+ rigorously fact-checked data points per state, covering everything from voter ID laws to voter registration deadlines.
 
 The software is free to use and embed on your website. If you'd like access to the data that comes in via a dedicated instance of our software, you'll need to sign up for a VoteAmerica+ account.  You'll also need to sign up for a VoteAmerica+ account if you'd like to use the Civic Data API.
-- [Click here for the embed code for the free tools](https://www.voteamerica.com/embeds/)
+- [Click here for the embed code for the free tools](https://www.voteamerica.org/embeds/)
```

Comment:
While we're here ... we've moved this page. It redirects, but we might as well update the link to go directly where we want it to go, which is: https://secure.voteamerica.org/free-software

Reply:
Done, thank you!